ui/app/search/results-dir.html

```diff
@@ -1,15 +1,23 @@
 <div class="panel panel-default" ng-repeat="result in results">
   <div class="panel-heading">
-    <h4><span class="pull-left badge ml-badge">ML 8</span><a ng-href="detail{{result.uri}}">{{result.metadata.name.values[0] || result.uri}}</a></h4>
+    <h4><span class="pull-left badge ml-badge">ML {{result.metadata.version.values[0] }}</span><a ng-href="detail{{result.uri}}">{{result.metadata.name.values[0] || result.uri}}</a></h4>
   </div>
   <div class="matches panel-body">
     <div ng-repeat="match in result.matches">
       <span ng-repeat="text in match['match-text'] track by $index">
         <span ng-if="text.highlight === undefined">{{text}}</span>
         <span class="highlight" ng-if="text.highlight !== undefined">{{text.highlight}}</span>
-        <span class="features"><strong>Features:</strong> list, of, features</span>
-        <span class="browsers"><img src="/images/icons/chrome.png"/><img src="/images/icons/firefox.png"/><img src="/images/icons/ie.png"/><img src="/images/icons/safari.png"/></span>
       </span>
     </div>
+      <div class="features">
+        <span class="features" ng-show="result.metadata.feature.values.length > 0"><strong>Features:</strong>
+        <span ng-repeat="value in result.metadata.feature.values">{{value}}<span ng-hide="$last">, </span></span>
+ </span>
+        <!-- <em>Features</em> -->
```

Comment:
I think these commented lines can be deleted. You can append to your pull request by committing and pushing more changes.